quickstarts/EAP7-2033_EAP8_GA_Quickstarts.adoc

```diff
@@ -0,0 +1,132 @@
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This proposal specifies which Quickstarts will be delivered on EAP 8 GA, and which Cloud, Provisioning and Automated Testing enhancements will be demonstrated by such Quickstarts.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/EAP7-2033[EAP7-2033]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1891[EAP7-1891]
+* https://issues.redhat.com/browse/EAP7-1792[EAP7-1792]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:szhantem@redhat.com[Sultan Zhantemirov]
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [x] QE
+
+=== Affected Projects or Components
+
+* https://github.com/wildfly/quickstart
+* https://github.com/jboss-developer/jboss-eap-quickstarts
+
+=== Other Interested Projects
+
+* None
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+
+* [x] Bootable jar (local and cloud deployment)
+* [x] OpenShift S2I
+
+== Requirements
+
+=== Hard Requirements
+
+* The Quickstarts to deliver with EAP 8 GA should:
+** Cover EAP 8 GA main functional areas, such as Web apps, Web Services/REST, Database/JPA, Messaging (including remote broker), Transactions and EJBs.
+** Be compatible with OpenShift.
+
+* The following EAP 8 Cloud/Provisioning/Testing enhancements should be delivered by at least one of the EAP 8 GA Quickstarts
+** Provisioning
+*** The WildFly Maven Plugin is now capable of provisioning a server with specific features (through Galleon layers), configure it using CLI scripts, and package it with an application deployed. Each EAP 8 Beta Quickstart should demonstrate how to use this new WildFly Maven Plugin functionality to provision a server with the needed Galleon layers, configure it using a CLI script if required, and package it with Quickstart’s application deployed.
+** Cloud
+*** Helm charts may be used to build and deploy an application on OpenShift, with improved user experience compared with the previous S2i usage. Each EAP 8 Beta Quickstart should demonstrate how to use Helm Charts to build and deploy the Quickstart’s application on OpenShift.
+** Testing
+*** The Quickstarts should demonstrate how users may test their applications with JBoss EAP 8, on baremetal and OpenShift environments. Regarding the JBoss EAP 8 Beta release the delivered Quickstarts should include integration tests, abstract from the environment where the server is running, and take advantage of Arquillian and the other Quickstarts enhancements to run such tests.
+**** On baremetal the WildFly Maven Plugin should be used to locally provision the server with the application deployed, and use the Arquillan managed container to run the tests.
```

Comment:
@emmartins since EAP won't support Maven Plugin to provision the server and will leverage the old-way method of starting the server, this sentence can be rewritten.

By the way, the `arq-managed` seems to be obsolete (https://issues.redhat.com/browse/WFLY-17823)

Thanks

Reply:
removed